### ISSUE TITLE
Add quantile normalization

### ISSUE THREAD

**Issue by floscha** [floscha/tabular-dataset#62]
Add a new `quantile` method to the `numeric.normalize` transformation, based on scikit-learn's 
[QuantileTransformer](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.QuantileTransformer.html)
